commands/config-get.md

```diff
@@ -6,23 +6,25 @@ SET`.
 
 `CONFIG GET` takes multiple arguments, which are glob-style patterns.
 Any configuration parameter matching any of the patterns are reported as a list
-of key-value pairs.
+of key-value pairs, sorted alphabetically by keys.
```

Comment:
Users refer to these docs even if they're running older versions, so we should mention in which versions this is sorted.

```suggestion
of key-value pairs.
In Valkey 9.0 and later, the list is sorted alphabetically by keys.
```